input/resources/codesystem-au-recorded-sex-or-gender-type.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="au-recorded-sex-or-gender-type" />
+    <meta>
+        <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem" />
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+        <valueInteger value="0"/>
+    </extension>
+    <url value="http://terminology.hl7.org.au/CodeSystem/au-recorded-sex-or-gender-type" />
+    <identifier>
+        <system value="urn:ietf:rfc:3986"/>
+        <value value="urn:oid:2.16.840.1.113883.2.3.4.1.4.29"/>
+    </identifier>
+    <name value="AURecordedSexOrGenderType" />
+    <title value="AU Recorded Sex or Gender Type" />
+    <status value="active" />
+    <experimental value="false"/>
+    <description value="Codes that represent the type of a Recorded Sex or Gender (RSG) value in an Australian context." />
+    <caseSensitive value="true" />
+    <hierarchyMeaning value="is-a"/>
+    <valueSet value="http://terminology.hl7.org.au/ValueSet/au-recorded-sex-or-gender-type"/>
+    <compositional value="false" />
+    <versionNeeded value="false" />
+    <content value="complete" /> 
```

Comment:
Codesystem.property is missing. Must be added to use 'notSelectable'. I recommend using the definition from FHIR code system https://build.fhir.org/ig/HL7/fhir-extensions/CodeSystem-obligation.html unless a suitable alternative to adopt is suggested.

Reply:
Added.